allauth/account/views.py

```diff
@@ -46,7 +46,8 @@
     sync_user_email_addresses,
     url_str_to_user_pk,
 )
-
+from django.core.validators import EmailValidator
+from django.forms import ValidationError
```

Comment:
The isort is failing on these two lines if you take a look at the logs:

```python
--- /home/runner/work/django-allauth/django-allauth/allauth/account/views.py:before	2023-09-03 10:56:32.162313
+++ /home/runner/work/django-allauth/django-allauth/allauth/account/views.py:after	2023-09-03 10:56:51.994464
@@ -2,6 +2,8 @@
 from django.contrib.auth import REDIRECT_FIELD_NAME
 from django.contrib.auth.decorators import login_required
 from django.contrib.sites.shortcuts import get_current_site
+from django.core.validators import EmailValidator
+from django.forms import ValidationError
 from django.http import (
     Http404,
     HttpResponsePermanentRedirect,
@@ -46,8 +48,7 @@
     sync_user_email_addresses,
     url_str_to_user_pk,
 )
-from django.core.validators import EmailValidator
-from django.forms import ValidationError
+
```

There should be two lines between the first code line and imports. You removed one.

 The imports should be moved to be sorted alphabetically, as can be seen in the patch by isort.

Reply:
Thanks @fdobrovolny for the suggestion, turns out the isort and black installation were messed up.